vowpalwabbit/core/src/reductions/cb/details/large_action_space.h

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) by respective owners including Yahoo!, Microsoft, and
+// individual contributors. All rights reserved. Released under a BSD (revised)
+// license as described in the file LICENSE.
+#pragma once
+
+#include "Eigen/Dense"
+#include "vw/core/action_score.h"
+#include "vw/core/rand48.h"
+#include "vw/core/v_array.h"
+#include "vw/core/vw_fwd.h"
+
+#include <iostream>
+#include <vector>
+
+namespace VW
+{
+namespace cb_explore_adf
+{
+struct cb_explore_adf_large_action_space
+{
+  uint64_t d = 0;
```

Comment:
Members should be prefixed with `_`

Reply:
members or private members?

Reply:
changed the private ones

Reply:
We weren't prescriptive but seems to imply all members.

> Class member variables should be prefixed with _. Example: uint32_t _number_of_actions;

https://github.com/VowpalWabbit/vowpal_wabbit/wiki/Coding-guidelines#style

Reply:
Yeah we aren't. I don't like accessing public data members via `data._Q` though that seems wrong. This struct is going to evolve a bit so I'll modify it as we go along